### PR TITLE
Auto registering log channel

### PR DESCRIPTION
**Pull request by HajMo**
This pull request will enable LaraBug without registering it manually as it's described in installation guide in project details page.

### CODE REVIEW

**Comment by Cannonb4ll** [comment]
What if users want to disable it?

**Comment by HajMo** [comment]
They can delete the `login_key` value in `.env` or we can add a new env variable to decide if the service is enabled.

**Comment by SebastiaanKloos** [comment]
@HajMo There should be an easy option to disable this feature as the October CMS plugin would break when merging this PR. Configuring config files is very personal and that's possibly why this does not exist in the package. @Cannonb4ll what do you think?

**Comment by Cannonb4ll** [comment]
Yes I agree, I'm not too sure about this PR, of course, it is nice, but I recon it'll do more harm rather than it being helpful.

**Comment by HajMo** [comment]
@SebastiaanKloos We can add more checks in the [condition](https://github.com/LaraBug/LaraBug/blob/e84470f5f9e0ee6bcb02c9747fe010c4b3c102c4/src/ServiceProvider.php#L72) with config option that decide if this feature is enabled, something like:
```php
if (config('larabug.login_key') && config('larabug.auto_register')) {
    // Register
}
```
And in `config/larabug.php` there's will be boolean `auto_register` variable. I think this will be a solution.

**Comment by Cannonb4ll** [comment]
I'm still quite unsure whether to merge this at this point.

**Comment by HajMo** [comment]
@SebastiaanKloos What you think?

**Comment by SebastiaanKloos** [comment]
> @SebastiaanKloos What you think?

As @Cannonb4ll already said, we are still unsure whether we are going to merge this.

Let's leave this PR open for now and see if more users want this merged.

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "larabug/larabug",
+    "name": "hajmo/larabug",
```
Comment (lloricode) [inline composer.json:2]: is this intended?

**Review by lloricode — CHANGES_REQUESTED** [review]
please update to `larabug/larabug`

**Comment by lloricode** [comment]
> Yes I agree, I'm not too sure about this PR, of course, it is nice, but I recon it'll do more harm rather than it being helpful.

You have point, let developer manage since, so developer will aware of configuration rather than register automagically

**Comment by SebastiaanKloos** [comment]
@HajMo I appreciate the work you've put in the PR. However, we are not going to implement this in the near future. 

I recommend forking the repo and adding your functionality, or creating your own package requiring the larabug/larabug package. Which automatically sets the logging configuration. 

@Cannonb4ll @Glennmen do you agree?

**Comment by Cannonb4ll** [comment]
Yes, for now I agree. Once this gets more replies, comments and genuine use-case examples I'd be glad to take another look at it.

**Pull request closed without merging**